reduce_geo.go

```diff
@@ -0,0 +1,44 @@
+package metrics
+
+import (
+	"strconv"
+
+	"github.com/mmcloughlin/geohash"
+)
+
+func convertLatLon(rawLat, rawLon string) (float64, float64, error) {
+	lat, err := strconv.ParseFloat(rawLat, 64)
+	if err != nil {
+		return 0.0, 0.0, err
+	}
+	lon, err := strconv.ParseFloat(rawLon, 64)
+	if err != nil {
+		return 0.0, 0.0, err
+	}
+	return lat, lon, nil
+}
+
+func reduceGeoHashLabels(labels map[string]string) map[string]string {
+	rawLat, ok := labels["lat"]
```

Comment:
Where are we setting up the `lat` , `lon` labels? I am only seeing `geo_hash` label being set up. Would love some pointers.